README.md

```diff
@@ -39,7 +39,7 @@ in ECE and CS at CMU.
 - 15-424
 - [15-440: Distributed Systems](electives/15440.md)
 - [15-455: Undergraduate Complexity Theory](electives/15455.md)
-- 16-385
+- [16-385: Computer Vision](electives/16385.md)
```

Comment:
Need to add links for 424 and 447

Might as well do this when 418 is also done as well.